src/app/models/AudioRecording.ts

```diff
@@ -131,6 +131,10 @@ export class AudioRecording
     return apiRoot + audioRecordingOriginalEndpoint(this.id);
   }
 
+  public getSplittableUrl(apiRoot: string): string {
```

Comment:
we use the term `media` to describe fragments of an audio recording file. 
```suggestion
  public getMediaUrl(apiRoot: string): string {
```